src/box/lua/config/applier/roles.lua

```diff
@@ -0,0 +1,75 @@
+local log = require('internal.config.utils.log')
+
+local last_loaded = {}
+
+local function apply(_config)
+    log.verbose('roles.apply: do nothing')
+end
+
+local function post_apply(config)
```

Comment:
`app` is loaded in `post_apply`, because we want the app code working in the `ready` config state with configuration accessible from `config:get()`.

Is the motivation for using `post_apply` the same for role's code? Should we consider a config as successfully applied before we're run role's code?

I have no objections, but I want to understand all the hidden details.

Reply:
I thought maybe the roles should also have access to the config. However, they have their own config, so I removed post_apply.